content/en/guides/hosting/monitoring-usage/org_dashboard.md

```diff
@@ -6,92 +6,92 @@ menu:
 title: View organization activity
 ---
 
-This page shows various ways to view activity within your W&B organization.
+This page describes how to view activity in your W&B organization. Select your deployment type to continue.
 
-## View user status and activity
+## View user status and activity 
 
 {{< tabpane text=true >}}
-{{% tab header="Dedicated / Self-managed" value="dedicated" %}}
-1. To access the **Organization Dashboard**, navigate to `https://<org-name>.io/org/dashboard/`. Replace `<org-name>` with your organization name. The **Users** tab opens by default. It lists all users, along with data about each user.
-1. To sort the list by user status, click the **Last Active** column label. Each user's status is one of the following:
-
-    * **Invite pending**: Admin has sent invite but user has not accepted invitation. 
-    * **Active**: User has accepted the invite and created an account.
-    * **-**: The user was previously active but has not been active in the last 6 months.
-    * **Deactivated**: Admin has revoked access of the user.
-1. To see details about a user's last activity, hover your mouse over the **Last Active** field for the user.  A tooltip appears that shows when the user was added and how many total days the user has been active.
-
-    A user is _active_ if they:
-    - log in to W&B.
-    - view any page in the W&B App.
-    - log runs.
-    - use the SDK to track an experiment.
-    - interact with the W&B Server in any way.
+{{% tab header="Dedicated Cloud / Self-Managed" value="dedicated" %}}
+1. Navigate to the **Organization Dashboard** at `https://<org-name>.io/org/dashboard/`. Replace `<org-name>` with your organization name. The **Users** tab opens by default and lists every user in the organization.
+2. To sort the list by user status, click the **Last Active** column header. Each user's status is one of the following:
+   * **Invite pending** – an invitation was sent but not yet accepted.
+   * **A timestamp** – the user accepted the invitation and has signed in at least once. The timestamp indicates the most recent activity.
+   * **–** – the user was previously active but has not been active in the last six months.
```

Comment:
Bolding a single dash here renders weird in the UI (see doc preview). Can you use words and then add in paranthesis what it'll look like in the UI?